components/tooltip/index.css

```diff
@@ -10,20 +10,78 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../overlay/index.css";
+@import "../commons/overlay-coretokens.css";
 
 .spectrum-Tooltip {
-  --spectrum-tooltip-neutral-target-offset: 3px;
-  /* This variable needs to be updated in Spectrum DNA. Using the height variable so it scales to spectrum-large */
-  --spectrum-tooltip-neutral-tip-width: var(--spectrum-tooltip-neutral-tip-height);
+  --spectrum-tooltip-animation-duration: var(--spectrum-animation-duration-100);
+
+  /* animation distance is equal to spacing to source */
+  --spectrum-tooltip-animation-distance: var(--spectrum-spacing-75);
+
+  /* override if additional spacing to source is required */
+  --spectrum-tooltip-margin: 0px;
+
+  --spectrum-tooltip-height: var(--spectrum-component-height-75);
+  --spectrum-tooltip-max-inline-size: var(--spectrum-tooltip-maximum-width);
+  --spectrum-tooltip-border-radius: var(--spectrum-corner-radius-100);
+
+  --spectrum-tooltip-icon-width: var(--spectrum-workflow-icon-size-50);
+  --spectrum-tooltip-icon-height: var(--spectrum-workflow-icon-size-50);
+
+  --spectrum-tooltip-font-size: var(--spectrum-font-size-75);
+  --spectrum-tooltip-line-height: var(--spectrum-line-height-100);
+  --spectrum-tooltip-cjk-line-height: var(--spectrum-line-height-cjk-100);
+  --spectrum-tooltip-font-weight: var(--spectrum-font-weight-regular); /* TODO - check is still 400 */
+
+  /* horizontal spacing */
+  --spectrum-tooltip-spacing-inline-start: var(--spectrum-component-edge-to-visual-75);
+  --spectrum-tooltip-spacing-inline-end: var(--spectrum-component-edge-to-text-75);
```

Comment:
This causes padding on the left of `7px` and padding on the right of `9px`, is that expected?

<img width="213" alt="image" src="https://user-images.githubusercontent.com/1156657/205688062-7424262d-10a8-4b87-8df7-589d357f24aa.png">

Reply:
@lunarfusion, FYI.